### ISSUE TITLE
cannot craft inside a crafting grid as it have no access to fluids storage

### ISSUE THREAD

**Issue by atothek1** [raoulvdberge/refinedstorage#1551]
#### Issue description:
If i want to craft something that needs fluids i cannot access my stored fluids within the crafting grid. I have to get me buckets out the Fluids Grids before.

#### What you expected to happen:
Be able to craft within the crafting grid with items and fluids.

#### Steps to reproduce:
1. open crafting grid
2. try to craft something that needs water, or search for fluids

- Minecraft: 1.12.2
- Forge: 14.23.0.25.37
- Refined Storage: 1.5.23

Does this issue occur on a server? [yes/no] not sure

**Comment by raoulvdberge** [comment]
Be more specific.  Do you mean crafting in the crafting grid or autocrafting?

**Comment by atothek1** [comment]
Hi Raoul, thanks for reply and this awesome mod.
I mean in the crafting grid, if i want do craft something that needs water or destabalized redstone and i have them in the system, the fluids are not available in the Crafting grid.

**Comment by raoulvdberge** [comment]
You are not answering my question...

**Comment by atothek1** [comment]
sorry more specific i cannot be, 
Crafting in the crafting grid with fluids from the fluid storage is not possible. The fluids are not showing up.

maybe its intended to be like this.

**Comment by raoulvdberge** [comment]
Let me rephrase: did you start an autocrafting task or did you craft manually?

**Comment by atothek1** [comment]
I want to craft manually :)

**Comment by raoulvdberge** [comment]
And you tried to get the buckets out of the fluid system by using JEI autocomplete (clicking on the "+") sign, right?

**Comment by atothek1** [comment]
I have them not in the crafting grid available, i have 10.000 mB in the storage, in the Fluid Grid all is fine,
than i switch to the Crafting Grid and the Fluids are not there. No autocomplete or anything. pure manually crafting :)

**Comment by raoulvdberge** [comment]
Well, obviously, fluids aren't shown in the crafting grid but in the fluid grid. You have to use 2 grids.
